documentation/docs/steps/neoDeploy.md

```diff
@@ -54,3 +54,50 @@ steps:
       account: <myDeployAccount>
       host: hana.example.org
 ```
+
+## Example for invalidating the cache
+
+* Setting parameter `invalidateCache` to `true`, will clean up the cache of a Fiori Launchpad site, refreshing the content of any applications deployed there. This is **applicable** only when the application is deployed and accessed via Portal service (Fiori Launchpad site).
+
+Setting this parameter to true, needs additional configurations to go through. Firstly, create an oauth credential as shown below:
+
+### OAuth credential creation
+
+  1 Select the oauth settings from your subaccount, create a new client with subscription to portal/nwc as shown in the following images:
+
+    ![OAuth client creation](../images/oauthClientCreation.png)
```

Comment:
It doesn't seem to find this image. [Check the resulting page here](https://github.com/SAP/jenkins-library/blob/12f450a55ee2131d9468aa6c371d02326704b6c3/documentation/docs/steps/neoDeploy.md#example-for-invalidating-the-cache).